### ISSUE TITLE
“onPointerMove” Method does not work

### ISSUE THREAD

**Issue by gk-1213** [benthillerkus/betrayal#30]
The onPointerMove method in this code does not work

```
import 'package:flutter/material.dart';
import 'package:betrayal/betrayal.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
        home: Scaffold(
            appBar: AppBar(title: const Text("Look at the system tray 👀")),
            body: Center(
                child: TrayIconWidget(
              winIcon: WinIcon.application,
              // tooltip: "Here I am!",
              onPointerMove: (position) {
                print(position);
              },
              child: Text("111"),
            ))));
  }
}
```

**Comment by benthillerkus** [comment]
Can you share some system information incl Windows version?

**Comment by gk-1213** [comment]
processor	12th Gen Intel(R) Core(TM) i5-12400   2.50 GHz
Belt RAM	     16.0 GB (15.7 GB 可用)
System type 64-bit operating system, x64-based processor
Pen and touch No pen or touch input available for this display

Version     Windows 11 Professional
Version     21H2
Operating system version	22000.1455

**Comment by benthillerkus** [comment]
Haven't had a chance to look at this (yet), but what happens when you enable the `BetrayalPluginLogger` and set it to `fine`? iirc it should first log that the event was registered and then whenever you hover the cursor over the icon it should log the position.

**Comment by gk-1213** [comment]
```
import 'package:flutter/material.dart';
import 'package:betrayal/betrayal.dart';

void main() {
  BetrayalLogConfig.allowIndividualLevels();
  BetrayalLogConfig.level = "FINER";
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
        home: Scaffold(
            appBar: AppBar(title: const Text("Look at the system tray 👀")),
            body: Center(
                child: TrayIconWidget(
              winIcon: WinIcon.application,
               onPointerMove: (position) {
                 print(position);
               },
              child:const Text("111"),
            ))));
  }
}
```



![image](https://user-images.githubusercontent.com/76909981/220225576-03ee887c-c70f-449a-a499-e01d82a23f9e.png)

**Comment by gk-1213** [comment]
Where should I go to see the log output you said？

**Comment by gk-1213** [comment]
I can see some information in Flutter's log when I open BetrayalPluginLogger.
![image](https://user-images.githubusercontent.com/76909981/220232201-cb48dcb7-bc67-46ea-bb1f-4e2c961fd86e.png)
But why doesn't the "print" method work in "onPointerMove"

**Comment by benthillerkus** [comment]
@gk-1213 both of your issues should be fixed in `betrayal 1.3.7`. Thanks for pointing them out!
